ci/test.sh

```diff
@@ -33,7 +33,7 @@ function main {
     mvn clean verify -q -f $REL/../rest/ -B -pl taskana-rest-spring-example -P history.plugin 
   elif [[ "$1" == "POSTGRES_10_4" ]]; then
     mvn clean verify -q -f $REL/../lib/taskana-core -B
-    mvn clean install -q -f $REL/../rest/ -B -P postgres
+    mvn clean install -q -f $REL/../rest/ -B -DskipTests -Dmaven.javadoc.skip=true
```

Comment:
Maybe we do not need this. But what is actually the problem here? We shoult at least try to understand it. This sounds like avoiding problems instead of solving them...

Reply:
The issue was resolved. This file wasn't modified